### DISCUSSION TITLE
Feature request: SUM()

### DISCUSSION THREAD

**Discussion by reqwire** [directus/directus#15561] [Archive]
EDIT: Nevermind: https://docs.directus.io/reference/query.html#aggregation-grouping
EDIT 2: Why are there no examples on how to use it

**Comment by azrikahar** [comment]
Having more examples is definitely something we'd like to add throughout the docs. Perhaps query parameters can also be fit into the overhaul of API reference as indicated over here: https://github.com/directus/docs/milestone/2, cc @erondpowell (provided it wasn't already included of course 😄)

**Comment by erondpowell** [comment]
@reqwire Hey there! What Azzy said is correct, we have a full update to the API docs in the works.

If I'm not mistaken, the example in the section you linked is universal, used for all aggregators. Including sum.

https://user-images.githubusercontent.com/62306630/190196685-2455cc52-4875-4a7f-88af-50e5b62b57a1.mov

**Reply by reqwire** [reply]
> @reqwire Hey there! What Azzy said is correct, we have a full update to the API docs in the works.
> 
> If I'm not mistaken, the example in the section you linked is universal, used for all aggregators. Including sum.
> 

Thanks, although I still haven't managed to get any to work other than count()
